nix/mobile/android/android-env.nix

```diff
@@ -25,7 +25,8 @@ let
     ndkVersion = "19.2.5345600";
     useGoogleAPIs = false;
     useGoogleTVAddOns = false;
-    includeExtras = [ "extras;android;m2repository" "extras;google;m2repository" ];
+    includeExtras =
+      [ "extras;android;m2repository" "extras;google;m2repository" ];
```

Comment:
Really dislike this format, looks like a syntax error to me. This formatter looks pretty inconsistet in how it handles arrays, for example look at `nix/desktop/macos/default.nix`.

Reply:
Yeah, I guess it is trying to make lines as short as possible. I agree with you that it still needs quite a bit of optimization (at the very least be able to turn on/off certain rules). I'll rollback these particular changes for now.